lib/Controller/PageController.php

```diff
@@ -72,4 +72,25 @@ public function context(int $contextId): TemplateResponse {
 
 		return $this->index();
 	}
+
+	#[PublicPage]
+	#[NoCSRFRequired]
+	#[OpenAPI(scope: OpenAPI::SCOPE_IGNORE)]
+	#[FrontpageRoute(verb: 'GET', url: '/s/{token}')]
+	public function linkShare(string $token): TemplateResponse {
+		Util::addScript(Application::APP_ID, 'tables-main');
```

Comment:
@enjeck may keep this if new functionality goes into the main script, or otherwise we can change this if course and load a different one.

The share token is being provided via initialState a few lines below.